js/adapt-contrib-tutor.js

```diff
@@ -9,25 +9,28 @@ class Tutor extends Backbone.Controller {
   initialize() {
     this.listenTo(Adapt, {
       'componentView:postRender': this.onComponentViewPostRender,
-      'questionView:showFeedback': this.onQuestionViewShowFeedback
+      'questionView:showFeedback': this.onQuestionViewShowFeedback,
+      'buttonsView:postRender': this.onButtonsViewPostRender
     });
   }
 
   onComponentViewPostRender(view) {
-    const model = view.model;
+    const { model } = view;
     if (!model.isTypeGroup('question')) return;
-    const config = model.get('_tutor');
-    if (!config) return;
-    const type = TUTOR_TYPE(config._type?.toUpperCase());
-    if (type !== TUTOR_TYPE.INLINE) return;
-    this.listenToOnce(Adapt, 'buttonsView:postRender', this.onButtonsViewPostRender);
-    if (model.get('_canShowFeedback') && model.get('_isSubmitted')) {
-      model.setupFeedback();
-      Adapt.trigger('questionView:showFeedback', view);
-    }
+    const shouldShowFeedback = (model.get('_canShowFeedback') && model.get('_isSubmitted'));
+    if (!shouldShowFeedback) return;
+    model.setupFeedback();
+    Adapt.trigger('questionView:showFeedback', view);
   }
 
   onButtonsViewPostRender(view) {
+    const { model } = view;
+    const config = (!model.get('_tutor') || model.get('_tutor')?._isInherited === true)
```

Comment:
```suggestion
    const config = (!model.get('_tutor') || (model.get('_tutor')?._isInherited ?? true))
```
If the `_tutor` config is present but empty, or missing `_isInherited` it wasn't defaulting to `_isInherited: true` as per the schema. Unlikely for someone to manually configure it this way, but if attributes are missing I guess they should use defaults instead. Should probably add this default value into [TutorModel](https://github.com/adaptlearning/adapt-contrib-tutor/blob/master/js/TutorModel.js) too.